build/compile/LzmaArchive.targets

```diff
@@ -17,5 +18,6 @@
                     ContinueOnError="WarnAndContinue" />
 
       <Copy SourceFiles="$(IntermediateArchive)" DestinationFiles="$(FinalArchive)" />
+      <Copy SourceFiles="$(IntermediateArchive)" DestinationFiles="$(PackagesArchive)" />
```

Comment:
Why do we need to copy it to yet another directory. This file isn't small. Can we just publish it from where it's already written?